### ISSUE TITLE
[Gossiptext] Grelag

### ISSUE THREAD

**Issue by Jiinkey** [NetherstormTeam/IssueTracker#317]
Grelag at Spinebreaker Post, Hellfire Peninsula is supposed to say:
"Thank you for your help in making the salve, *name*. Once it's had a chance to dull the pain, I'm going to see how far I can get on this leg."

Screenshot: http://i.imgur.com/dyq4S2S.jpg

Coordinates: X: -1329.096191 Y: 2392.082031 Z: 88.947670
NPC Info: Entry: 16858, GUID: 58047, Name: Grelag

Note: Currently says "Greetings name"

**Comment by Sdyessdev** [comment]
Is this only after completing a quest?

**Comment by Jiinkey** [comment]
After completing "Preparing the Salve" (http://tbc.openwow.com/quest=9345) this is what his gossip text should be (the one in the OP).
Before completing "Preparing the Salve" he should say:
"I've managed to work myself into quite a spot here. Far Seer Regulkut tried to warn me, but my carelessness has finally caught up with me. 

I came down here to study the ruins of Zeth'Gor, but ran afoul of the Legion on my way. Now, all I have to show for it is a nasty gash on my leg."


Screenshot: http://i.imgur.com/nV0uTcy.jpg
